docs/tools.md

```diff
@@ -379,6 +379,10 @@ const tool = new DynamicTool({
 
 _Source: [examples/tools/custom/dynamic.ts](/examples/tools/custom/dynamic.ts)_
 
+The name of the tool is required and must only contain characters between
+a-z, A-Z, 0-9, or one of - or \_.
+The inputSchema and description are also both required.
```

Comment:
Please highlight `inputSchema` and `description`.

Reply:
@Tomas2D updated as requested